VenueNextSDK-Demo/Controllers/DeepLinkTableViewController.swift

```diff
@@ -248,7 +248,7 @@ class DeepLinkTableViewController: UITableViewController {
                         self.view.isUserInteractionEnabled = false
                     }
                     
-                    VenueNext.shared.initialize(sdkKey: AppDelegate.sdkKey, sdkSecret: AppDelegate.sdkSecret, externalID: externalID, configURL: configURL) { (success, error) in
+                    VenueNext.shared.initialize(sdkKey: "SDK_KEY", sdkSecret: "SDK_DELEGATE", externalID: externalID, configURL: configURL) { (success, error) in
```

Comment:
For here because i added a `re-init` button for QA to test that needs the keys, we will have to store the key in a central place like i threw 2 static vars on the AppDelegate.

Reply:
The branch didn't have the static vars tho.

Reply:
Ok ill add them back